### ISSUE TITLE
Doc site on readthedocs?

### ISSUE THREAD

**Issue by fralau** [DP-3T/dp3t-sdk-ios#117]
I could not find a website with documentation. 

Do you intend to publish one in the next days?

**Comment by UBaggeler** [comment]
The documentation of SwissCovid and the respective backend systems can be found here: https://github.com/admin-ch/PT-System-Documents

**Comment by fralau** [comment]
Issues have been raised concerning the completeness, accuracy and usefulness of that documentation.

"The source codes have nearly no comment at all and we could not find thepromised technical specifications. We hope it will code in the future.16https://github.com/google/exposure-noti cations-server/blob/master/docs/design/veri cationprotocol.md17https://www.admin.ch/gov/fr/accueil/documentation/communiques.msg-id-79103.html18The source codes and technical speci cations of all components are publicly available.8

The repository of the DP3T project contains lots of source codes (in addition to the aboveones). Documents [5] describe the DP3T scheme but not the SwissCovid one. Actually, most of the DP3T scheme is replaced by GAEN and the part of the architecture related to theserver is nearly not covered at all.The repositories of theGAENAPI contains a sample code for an app and for a server.The GAEN API has no source code available but is fully documented."

[Prof. Vaudenay's report](
https://lasec.epfl.ch/people/vaudenay/swisscovid.pdf), pages 8 and 9.

Are there any plans by the EPFL team or Ubique make an effort to improve the documentation?

**Comment by UBaggeler** [comment]
We are actively working on improving the documentation of SwissCovid and the DP3T-SDK. Please see https://github.com/admin-ch/PT-System-Documents to get a better overview over the whole architecture (also server) and how EN is used. If this does not answer your questions, let us know.

**Comment by fralau** [comment]
Thanks a lot for these documents, which are interesting.

We would like to have "hands-on" documentation. Particularly, step-by-step procedures, that describe how to independently compile and test both the back-end and client apps (android and IOS), so as to perform tests.

**Comment by UBaggeler** [comment]
Yes, all necessary parts are available here on Github. The README of each repository should tell you how to compile and run it. If you've got specific questions, we will try to answer them (and improve the README).

**Comment by fralau** [comment]
I am aware of that, but I have not found instructions on how to finds and integrate the SDKs of Google and Apple (which I guess is free and easy to get, since the software is open source and open to scrutiny). 

In general, could give us a link to a clear, step-by-step how-to from A to Z? So that a person moderately skilled could do that?
